roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
Yes, I borrowed the code from openshift_logging_elasticsearch/tasks/main.yaml.  Are you suggesting to share the elasticsearch task with the mux (instead of having its own in the mux tasks/main.yaml)?